docs/source/airports/bucharest-tma/LROP.rst

```diff
@@ -1,31 +1,198 @@
+| Stand   | Aircraft    | Remarks       |
++---------+-------------+---------------+
+| 201     | B767/A31    | pushback only |
++---------+-------------+---------------+
+| 201A    | B744        | start-up      |
++---------+-------------+---------------+
+| 202-205 | B773        | pushback only |
++---------+-------------+---------------+
+| 206-217 | B757/A321   | pushback only |
++---------+-------------+---------------+
+| 218-223 | regional    | pushback only |
++---------+-------------+---------------+
+
+**Apron 3**
+
++--------+------------+----------------------------+
+| Stands | Aircraft   | Remarks                    |
++--------+------------+----------------------------+
+| all    | GA, bizjet | start-up at own discretion |
++--------+------------+----------------------------+
+
+Stand assignments by company:
+
++---------+----------------------------------------+---------------+
+| Stand   | Company                                | Aircraft      |
++---------+----------------------------------------+---------------+
+| 101-105 | LH Group: DLH, SWR, GWI, EWG, BTI, AUA |               |
++---------+----------------------------------------+---------------+
+| 105-106 | KLM                                    |               |
++---------+----------------------------------------+---------------+
+| 106     | THY                                    | A320/B738     |
++---------+----------------------------------------+---------------+
+| 107,109 | THY, UAL, AAL                          | A333/B77W     |
++---------+----------------------------------------+---------------+
+| 110     | ELY (default)                          |               |
++---------+----------------------------------------+---------------+
+| 113-115 | AFR, KLM                               |               |
++---------+----------------------------------------+---------------+
+| 114     | ACA, ROU, UAL, AAL                     | B767          |
++---------+----------------------------------------+---------------+
+| 201-205 | CARGO: BCS, SWN, CGF, ATG, BOX, DHK,   |               |
+|         | GEC, NPT, TAY, MNB, GCL, FRF           |               |
++---------+----------------------------------------+---------------+
+| APRON 3 | TIH                                    |               |
++---------+----------------------------------------+---------------+
+
+Regional: SF34, SB20, ATR, CRJ, SU95, A318
+
+LVP
+"""
+
+RWY 08R is authorized for CAT IIIA operations (RVR not less than 200 m) and guided take-off. RWY 08R is the main runway used during LVP.
+
+RWY 08L is authorized for CAT II operations (RVR not less than 350 m) and guided take-off.
+
+RWY 26L/R are authorized for CAT I operations and LVTO.
+
+If available visual aids do not provide sufficient taxi information, pilots may ask for Follow-me car guidence.
+
+Traffic regulation
+""""""""""""""""""
+
+In real life, LROP airport is closed for VFR traffic.
+
+On VATSIM, VFR traffic is permitted to fly at LROP. 
```

Comment:
`On VATSIM, VFR traffic is permitted to fly at LROP, only if IFR workload permits.` what do you think?